RiotSwiftUI/target.yml

```diff
@@ -53,6 +53,7 @@ targets:
     - path: ../Riot/Categories/UIColor.swift
     - path: ../Riot/Categories/UISearchBar.swift
     - path: ../Riot/Categories/UIView.swift
+    - path: ../Riot/Categories/MXSession.swift
```

Comment:
I don't think we should be doing this. The SwiftUI target is meant to be decoupled from the SDK.

Reply:
You're right I will remove it from the target it should used by services only.